core/templates/dev/head/profile/profile.html

```diff
@@ -33,8 +33,15 @@
   <div class="oppia-profile-container" ng-controller="Profile">
     <md-card class="oppia-profile-user-card">
 
+    <div ng-if=" '{{username}}' == '{{PROFILE_USERNAME}}' ">
```

Comment:
This, and the following lines, should be indented by 2, since they are all child nodes of md-card.